test/models/inbox_notice_test.rb

```diff
@@ -12,19 +12,22 @@ def notification
   end
 
   test 'requires a level' do
-    skip "One of: :info, :warn, :success, :danger"
+    skip 'One of: :info, :warn, :success, :danger'
   end
 
   test 'requires a state' do
-    skip "One of: :pending, :sent, :delivered, :unread, :read"
+    skip 'One of: :pending, :sent, :delivered, :unread, :read'
   end
 
   test 'notifications' do
-    skip "Notification event log, like NotificationEvent :sms, :number, :sent_at"
+    skip """
```

Comment:
Combine "" and "\n      Notification event log, like NotificationEvent\n      :sms, :number, :sent_at\n    " into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Combine "\n      Notification event log, like NotificationEvent\n      :sms, :number, :sent_at\n    " and "" into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.